edit/source-editor.js

```diff
@@ -95,6 +95,11 @@ function createSourceEditor(style) {
     dirty.clear();
   }
 
+  function updateStyleMeta(_style) {
```

Comment:
our `eslintrc` specifies _ as an indicator to suppress the warning about an unused parameter. What about `newStyle` here?